src/routes/__tests__/home.test.js

```diff
@@ -0,0 +1,32 @@
+const home = require("../home");
+
+describe("Test data", () => {
+  const dataOriginal = {
+    calendar: [
+      {
+        name: "Webkurs",
+        date: "13.12.2018",
+        time: "18:00-20:00",
+        eventlink: "https://www.facebook.com/pg/dielernwerkstatt/events"
+      }
+    ]
+  };
+
+  const stringifyData = JSON.stringify(dataOriginal);
```

Comment:
stringifyData sounds like a function name for me. Maybe stringif**ied**Data is better?

Reply:
Or something else, which is a noun. Variables should be nouns or adjectives/adverbs. Functions should be verbs.